.github/workflows/smoke-tests-on-demand.yaml

```diff
@@ -0,0 +1,182 @@
+name: On-Demand Smoke Tests
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  build:
+    if: |
+      github.event.label.name == 'run-smoke-tests' ||
```

Comment:
Nice, we should add this to docs I think - I added notes here:

https://github.com/datarobot-oss/cli/blob/main/docs/development/building.md#running-smoke-tests-using-github-actions